libs/FScreen.c

```diff
@@ -83,7 +83,7 @@ monitor_new(void)
 {
 	struct monitor	*m;
 
-	m = calloc(1, sizeof *m);
+	m = fxcalloc(1, sizeof *m);
 	//m->si = calloc(1, sizeof(struct screen_info));
```

Comment:
This change isn't correct, @klebertarcisio -- you need to update your copy of `master` in your repository first, then rebase this change on to it.

Reply:
Hi @ThomasAdam, thanks for the review.
 
Why is this change incorrect? My master is up to date. In the last commit, you changed the function **screen_info_new**, but in this one I'm changing the function **monitor_new**. Do they both need changes?

Reply:
Hi @klebertarcisio,

Ah, that's my mistake, sorry.  I had misread the change and was confused by the comment which doesn't need to be there, hence the following:

```suggestion
```

This change looks good.